api/tests/functional-tests/backend/core/test_geometry.py

```diff
@@ -92,3 +213,99 @@ def test_convert_geometry(
             model=None,
         )
     assert "currently unsupported" in str(e)
+
+
+def _load_polygon(db: Session, polygon) -> Polygon:
+    geom = json.loads(db.scalar(func.ST_AsGeoJSON(polygon)))
+    return schemas.metadata.geojson_from_dict(data=geom).geometry()
+
+
+def _load_box(db: Session, box) -> BoundingBox:
+    return schemas.BoundingBox(polygon=_load_polygon(db, box).boundary)
+
+
+def test_convert_from_raster(
+    db: Session,
+    create_objdet_dataset: str,
+    bbox: BoundingBox,
+    polygon: Polygon,
+):
+    annotation_id = db.scalar(
+        select(models.Annotation.id).where(
+            and_(
+                models.Annotation.box.is_(None),
+                models.Annotation.polygon.is_(None),
+                models.Annotation.multipolygon.is_(None),
+                models.Annotation.raster.isnot(None),
+            )
+        )
+    )
+    assert annotation_id is not None
+
+    q = _convert_raster_to_box([])
+    db.execute(q)
+
+    q = _convert_raster_to_polygon([])
+    db.execute(q)
+
```

Comment:
should we add a test for _convert_raster_to_multipolygon?

Reply:
Raster -> Multipolygon is not implemented as it is a non-trivial conversion. 

See #470 for reference.